### DISCUSSION TITLE
CodeRabbit found a bug in my `bru` file :)

### DISCUSSION THREAD

**Discussion by rabestro** [usebruno/bruno#1343] [General]
The CodeRabbit AI service discovered a bug in my `bru` file when I created a new request by copying and pasting an existing URL.

I am currently trying out different tools for API testing. For this purpose, I have created a repository that contains a sample REST service. Furthermore, I connected a new service called CodeRabbit for experimentation. CodeRabbit automatically performs code reviews and much to my surprise, it was able to detect errors in Bro files. Even more interestingly, the service offered me an automatic correction for my error. I have not corrected it yet so that I can show you the pull request.

Link to pull-request: 
https://github.com/rabestro/pig-latin-rest/pull/38

This service is available for free to open-source projects. Would it be helpful for Bruno's project?

<img width="852" alt="image" src="https://github.com/usebruno/bruno/assets/52966251/9f0a5cc1-0aa1-4616-a209-d0165056137a">

**Comment by helloanoop** [comment]
Wow! This is great!

Impressed by the AI review.
AI code review bots to perform an initial review clearly seems to be the future.
